basics/indexing/dictionary-index.md

```diff
@@ -0,0 +1,124 @@
+| [H3 (or geospatial)](geospatial-support.md) |                           | Requires no dictionary.                                             |
+
+
+## Configuration
+
+### How to enable or disable dictionaries
+Unlike most indexes, dictionary indexes are enabled by default, assuming that the number of unique values will
+be orders of magnitude smaller than the number of rows.
+
+If this is not the case, it is recommended to disable the dictionary for the column.
+In order to that, use the `disabled` property in `indexes.dictionary`:
+
+{% code title="Configured in tableConfig fieldConfigList" %}
+```javascript
+{
+  "fieldConfigList": [
+    {
+      "name": "theTableName",
+      "indexes": {
+        "dictioanry": {
+          "disabled": true
+        }
+      }
+    },
+    ...
+  ],
+...
+}
+```
+{% endcode %}
+
+Alternatively, the `encodingType` property can be changed. For example:
+```javascript
+{
+  "fieldConfigList": [
+    {
+      "name": "theTableName",
+      "encodingType": "RAW"
+    },
+    ...
+  ],
+...
+}
+```
+{% endcode %}
+
+Use the option you prefer, but try to be consistent, as Pinot will reject table configurations where the same column and
+index is defined in different places.
+
+The decision of actually create the dictionary or not can be delegated to some included Pinot heuristics.
+This feature can be enabled in `indexingConfig` object inside table config.
+The parameters that control the heuristic are:
+
+
+| Parameter                      | Default | Description                                                          |
+|--------------------------------|---------|----------------------------------------------------------------------|
+| optimizeDictionary             | false   | Enables the heuristic for all columns and activate some extra rules. |
+| optimizeDictionaryForMetrics   | false   | Enables the heuristic for metric columns.                            |
+| noDictionarySizeRatioThreshold | 0.85    | The ratio used in the heuristics.                                    |
+
+It is important to note that these parameters are configured for all columns on the table.
+When this optimization is enabled, columns explicitly marked as dictionary will actually be encoded as raw 
+if:
+- They are not multi-valued.
+- And their column type is fixed size (like int, long, double, timestamp, etc).
+- And their ratio between the forward index size encoded as raw and the forward index size encoded as dictionary is lower than `noDictionarySizeRatioThreshold`.
+
+If `optimizeDictionary` is false and `optimizeDictionaryForMetrics` is true, only columns declared as metric will be affected.
+If `optimizeDictionary` is true, all columns will be affected.
+Also in this case columns indexed as [text](./text-search-support.md) or [JSON](./json-index.md) will automatically be indexed as raw.
```

Comment:
in the case optimizeDictionary is true or false?

Reply:
In the case `optimizeDictionary` is true. I think the text is clear, but I'm open to a rewrite alternative if you find it confusing.